quarkus-workshop-super-heroes/docs/src/docs/asciidoc/core-appendix/appendix-installing-jq.adoc

```diff
@@ -0,0 +1,62 @@
+[[appendix-installing-curl]]
```

Comment:
Looks like the filenames got swapped for the curl and jq content, so `appendix-installing-jq` has the curl content.